docs/data/data_locations.md

```diff
@@ -0,0 +1,70 @@
+```
+
+**The primary purpose of this directory is to store small files, like github repositories and other code files.** Any scientific data you are using most likely belongs in a cloud bucket. Reserve the User Directory for scripts that load from and save data to these other storage locations.
+
+```{warning}
+To accommodate the expanding LEAP community, the data and compute team has instituted a storage quota on individual user directories `/home/jovyan`. Your home directory is intended only for notebooks, analysis scripts, and small datasets (< 1 GB). It is not an appropriate place to store large datasets. Unlike the cloud buckets, these directories use an underlying storage with a rigid limit. If a single user fills up the space, the Hub crashes for everyone. We recommend users use less than 25GB and enforce a hard limit of 50GB. **Users who persistently violate the limit may temporarily get reduced cloud access**.
+
+To check how much space you are using in your home directory open a terminal window on the hub and run `du -h --max-depth=1 ~/ | sort -h`.
+
+If you want to save larger files for your work use our [](reference.infrastructure.buckets) and consult our [Hub Data Guide](guide.data). See the [FAQs](faq.usr_dir_usage_warning) for guidance on reducing storage.
+```
+
+# Cloud Storage
+
+Our platform is centered around cloud data. LEAP owns two Google Cloud buckets (leap-persistent and leap-scratch) and also has an allocation of storage on an OSN pod.
+
+1. For "internal" use, i.e. during the research process, we recommend users use the JupyterHub in conjunction with LEAP Cloud buckets.
+1. For publishing or sharing data externally (i.e. in conjunction with a released paper), we recommend pushing to OSN Pods.
+
+## LEAP-Pangeo Cloud Storage Buckets
+
+LEAP-Pangeo provides users two cloud buckets to store data. Your [](reference.infrastructure.hub.server) is automatically authenticated to read from any of these buckets but write access might differ (see below). See [](guides.data.external.authentication) for details on how to access buckets from 'outside' the JupyterHub.
+
+- `gs://leap-scratch/` - Temporary Storage deleted after 7 days. Use this bucket for testing and storing large intermediate results. [More info](https://docs.2i2c.org/user/topics/data/cloud/#scratch-bucket)
+- `gs://leap-persistent/` - Persistent Storage. Use this bucket for storing results you want to share with other members.
+- `gs://leap-persistent-ro/` - Persistent Storage with read-only access for most users.
+
+Files stored on each of those buckets can be accessed by any LEAP member, so be conscious in the way you use these.
+
+- **Do not put sensitive information (passwords, keys, personal data) into these buckets!**
+- **When writing to buckets only ever write to your personal folder!** Your personal folder is a combination of the bucketname and your github username (e.g. \`gs://leap-persistent/funky-user/').
+
+## Open Storage Network (OSN) Pod
```

Comment:
I propose that we remove mention of the OSN Pod for the time being since most users won't have access. @norlandrhagen ?